sky/setup_files/dependencies.py

```diff
@@ -43,6 +43,7 @@
     'requests',
     'fastapi',
     'uvicorn[standard]',
+    'websockets',
```

Comment:
Since this is only used by kubernetes, how about we move it to L135, i.e. within the kubernetes dependencies?